README.md

```diff
@@ -25,7 +25,7 @@ In your app/build.gradle file, add Fidel dependency
 
 ```java
 dependencies {
-    implementation 'com.github.FidelLimited:android-sdk:1.7.5'
+    implementation 'com.github.FidelLimited:android-sdk:2.0.0'
```

Comment:
We should remove all docs from this README and point to the website. This was part of the Acceptance Criteria of [this ticket](https://fidelapi.atlassian.net/browse/CLET-152), but unfortunately this was not verified before the ticket was moved to DONE.